raycicmd/make.go

```diff
@@ -61,6 +62,20 @@ func parsePipelineFile(file string) (*pipelineGroup, error) {
 	return g, nil
 }
 
+func getTagFilter(config *config) ([]string, error) {
```

Comment:
maybe change to `runTagFilterScript()` (go normally do not write functions that start with `get`).

and please just send in the script command. don't send in the entire config.